controllers/auth.js

```diff
@@ -29,6 +29,9 @@ const githubAuth = (req, res, next) => {
         tokens: {
           githubAccessToken: accessToken,
         },
+        roles: {
+          archived: false,
+        },
```

Comment:
In my opinion, we can put this `roles` object somewhere else. Not sure where, but yes I don't think this is the appropriate place to put this `roles` object.
Can you find some other place where we can add this object to the user document ??

Reply:
Should I add it here ? Inside addOrUpdate model function
https://github.com/Real-Dev-Squad/website-backend/blob/15e6242972b7f674af869b618ed42a1c5ae080fd/models/users.js#L45

Reply:
Yes, I think we can put it here

Reply:
If we move this logic in users model's addOrUpdateUser method it'll break test cases where the fixtures already have a roles object

Reply:
As discussed with @ankushdharkar , a user should not have roles when it's added in firestore for the first time. For breaking tests we need to fix the fixtures and setup part of those tests